website/content/api-docs/system/internal-counters.mdx

```diff
@@ -94,11 +94,14 @@ period, which is represented by the `start_time` and `end_time` parameters.
 
 There are a few things to keep in mind while using this API.
 
-- The activity information only accounts for the activity of the latest
-contiguous months in the billing period. For example, if the billing period is
-from Jan 2022 to June 2022, and the activity subsystem in Vault was not
-capturing data for the months Jan to March, the response will only include
-information for May and June.
+- For vault versions before 1.11, the activity information only accounts for 
```

Comment:
small nit: should `vault` be capitalized here?

Reply:
Yes, thanks for catching!